### DISCUSSION TITLE
v2/zh/guide/markdown/vue-playground

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#349] [Announcements]
# v2/zh/guide/markdown/vue-playground

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.github.io/v2/zh/guide/markdown/vue-playground.html

**Comment by itzhameng** [comment]
大佬 有没有除了VUE可以交互，其他语言可以交互吗？例如 java,python
